pkg/volume/util/volumepathhandler/volume_path_handler.go

```diff
@@ -30,10 +30,8 @@ import (
 )
 
 const (
-	losetupPath           = "losetup"
-	statPath              = "stat"
-	ErrDeviceNotFound     = "device not found"
-	ErrDeviceNotSupported = "device not supported"
```

Comment:
storageos/storageos_util.go is only place which uses the same name, but it contains its own.
It is fine to remove this here.